MapboxCoreNavigation/RouteOptions.swift

```diff
@@ -9,4 +9,31 @@ extension RouteOptions {
             return .automotiveNavigation
         }
     }
+    
+    /**
+     Returns an optimal `RouteOptions` for navigation.
+     */
+    public convenience init(forNavigationWithWaypoints waypoints: [Waypoint], profileIdentifier: MBDirectionsProfileIdentifier? = .automobileAvoidingTraffic) {
+        self.init(waypoints: waypoints, profileIdentifier: profileIdentifier)
+        
+        includesSteps = true
```

Comment:
Why not subclass RouteOptions as NavigationOptions? Then NavigationOptions could store its own properties. This approach is intended to be supported by MapboxDirections.swift, and it has the benefit of being more discoverable and more easily documentable.